.github/workflows/rock-update.yaml

```diff
@@ -21,6 +21,10 @@ on:
         default: false
         required: false
         type: boolean
+      update-script:
+        description: "Custom script to update external dependencies in rockcraf.yaml"
```

Comment:
```suggestion
        description: "Custom script to update external dependencies in rockcraft.yaml"
```

Reply:
Script or command?

Reply:
Script; you can have a look [here](https://github.com/canonical/prometheus-rock/pull/14/files#diff-4cb9ac7d93a1e182e28baa0a65904ed264bc82c974a7b33d7132c53150b556bdR15-R26) to see how it is used